apps/sms/test/marionette/mocks/mock_message_thread.js

```diff
@@ -0,0 +1,38 @@
+module.exports = {
```

Comment:
> Injects: At first, I thought only mock_test_storage.js was needed for generating a thread, and the test didn't run. I thought abstracting it behind a function would make the intent clearer: If you want to generate a thread, you need these and only these 2. Do you think I should remove the abstraction?

```mock_test_storage.js``` is not only about threads, it's storage for every mock that needs storage adjustable from the test. In my WIP I use for mozContacts as well.

So when we need only contacts (for composer recipients tests for example) we don't need ```ThreadsMock```,  but we need storage and other mocks to be injected, and vice versa.

To summarize I think we don't need that abstraction, IMO having ```var MOCKS = [....]``` at the beginning of suite is more than enough - it's more like suite-level dependencies. At least how we do it in SMS.